components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/java/org/wso2/carbon/identity/api/server/identity/governance/v1/core/ServerIdentityGovernanceService.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, WSO2 LLC. (http://www.wso2.com).
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Why we are updating the date here?

Reply:
This needs to change as 2019-2024 as we include year range in the header. Made the change with https://github.com/wso2/identity-api-server/pull/621/commits/d093e671bebf98155c93db691775871d465a82bb